sdk/eventhub/azure-eventhub/samples/async_samples/sample_code_eventhub_async.py

```diff
@@ -79,11 +77,10 @@ def example_create_async_eventhub_consumer_client():
 
 
 async def example_eventhub_async_send_and_receive():
-    producer = example_create_async_eventhub_producer_client()
-    consumer = example_create_async_eventhub_consumer_client()
+    producer: EventHubProducerClient = example_create_async_eventhub_producer_client()
+    consumer: EventHubConsumerClient = example_create_async_eventhub_consumer_client()
```

Comment:
since both of these functions definitely return only the producer client, I don't believe these vars need annotations.